### PR TITLE
bazarr: 1.1.1 -> 1.1.2

### PR DESCRIPTION
**Pull request by andreisergiu98**
###### Description of changes

Updated bazarr
Closes https://github.com/NixOS/nixpkgs/pull/196226

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by NULLx76 — APPROVED** [review]

**Review by NULLx76 — COMMENTED** [review]
The commit message for 461470b9f3a6ed65fd901549e36ef36683957a80, should probably be prefixed with something like `nixos/tests/bazarr:` or just squashed into the other commit

**Comment by andreisergiu98** [comment]
@NULLx76 done

**Comment by 06kellyjac** [comment]
As an aside: I use [unrar-wrapper](https://search.nixos.org/packages?channel=unstable&show=unrar-wrapper&from=0&size=50&sort=relevance&type=packages&query=unrar) which has the free `unar` stand in place for the unfree `unrar` and it seems to work fine

`bazarr = prev.bazarr.override { unrar = prev.unrar-wrapper; };`

Probably not something we should do by default until the bazarr maintainers say it's supported but since unrar being unfree was mentioned I thought I'd post this :)

**Comment by bjornfor** [comment]
If you add `time.timeZone = "UTC";` to the nixos test configuration the test also *pass*. Apparently bazarr tries to use UTC itself when no timezone is given, but it fails. Explicitly setting the timezone seems to work.

**Comment by bjornfor** [comment]
> If you add `time.timeZone = "UTC";` to the nixos test configuration the test also _pass_. Apparently bazarr tries to use UTC itself when no timezone is given, but it fails. Explicitly setting the timezone seems to work.

I made an upstream issue: https://github.com/morpheus65535/bazarr/issues/1983

**Comment by andreisergiu98** [comment]
So should we add `time.timeZone = "UTC"` to the test or should we ignore the errors until it is fixed upstream?
Regarding `unrar`, bazarr 1.1.3 added support for `unar`, so I think it's safer to switch to it when it's released.

**Comment by bjornfor** [comment]
> So should we add time.timeZone = "UTC" to the test or should we ignore the errors until it is fixed upstream?

I would integrate that fix into the/a "nixos/tests/bazarr: fix it" commit, then do the version bump.

**Comment by bjornfor** [comment]
> I would integrate that fix into the/a "nixos/tests/bazarr: fix it" commit, then do the version bump.

I did that here: https://github.com/NixOS/nixpkgs/pull/199517

**Comment by andreisergiu98** [comment]
Cool! After you merge it, I'll drop the commit on the test and leave only the bump.

**Comment by bjornfor** [comment]
> Cool! After you merge it, I'll drop the commit on the test and leave only the bump.

Merged.

**Review by bjornfor — APPROVED** [review]
Rebased on master and `nix-build ./nixos/tests/bazarr.nix` passed.

**Pull request merged**
